dask/dataframe/tests/test_dataframe.py

```diff
@@ -3740,3 +3739,22 @@ def test_dtype_cast():
     assert ddf.B.dtype == np.int64
     # fails
     assert ddf.A.dtype == np.int32
+
+
+@pytest.mark.parametrize("base_npart", [1, 4])
+@pytest.mark.parametrize("map_npart", [1, 3])
+@pytest.mark.parametrize("sorted_index", [False, True])
+def test_series_map(base_npart, map_npart, sorted_index):
+    base = pd.Series([''.join(np.random.choice(['a', 'b', 'c'], size=3)) for x in range(100)])
+    if not sorted_index:
+        index = np.arange(100)
+        np.random.shuffle(index)
+        base.index = index
+    map_index = [''.join(x) for x in product('abc', repeat=3)]
+    map = pd.Series(np.random.randint(50, size=len(map_index)))
```

Comment:
`map` is a builtin, so lets call this `mapper`.

And pass `index=map_index` rather than assigning next.

Reply:
done